### DISCUSSION TITLE
Idle Current Tile Indicator Color Change

### DISCUSSION THREAD

**Discussion by connortrev** [runelite/runelite#16308] [Ideas]
The Skilling Notifications plugin has a notification overlay that tints your game window a certain color when you character goes idle. This is nice, but a bit annoying when switching between more attentive gameplay and more afk activities. 

It would be nice if the current tile indicator color (like from the Tile Indicators plugin) could change its color to when character goes idle. This would be a less obtrusive, but still obvious visual cue for idle.

**Comment by Nightfirecat** [comment]
Support for hub plugins should be directed to the author of the plugin.
You can find the support link by searching for the plugin in the Plugin Hub panel and clicking the `?` button on the plugin, or by right-clicking the plugin in the plugin panel and clicking the `Support` menu option.
